samples/dashboards/demo/cloud-monitoring/demo.css

```diff
@@ -0,0 +1,144 @@
+@import url("https://code.highcharts.com/css/highcharts.css");
+@import url("https://code.highcharts.com/datagrid/css/datagrid.css");
+@import url("https://code.highcharts.com/dashboards/css/dashboards.css");
+
+.highcharts-description {
+    padding: 0 20px;
+}
+
+.cloud-monitoring-data-controls {
+    background-color: var(--dashboard-bck-gray);
+    border-bottom: 1px solid #a5abb1;
+    padding: 20px 15px;
+}
+
+:root,
+.highcharts-light {
+    /* Colors for data series and points */
+    --highcharts-color-0: #33a29d;
+    --highcharts-color-2: #fe9d00;
+
+    /* Extra colors */
+    --bck-gray: #f1f5f9;
+    --dashboard-bck-gray: #e4eaf1;
```

Comment:
1. Do we need two variables here for almost the same color?
2. IMO it is not good practice to name variables in CSS by specifying a color in its name. Better to use something like `--bcg-color`.
3. I think that this color should have a darker counterpart in dark mode (e.g. `#252525`). A light background and dark components create too much contrast.